style.css

```diff
@@ -0,0 +1,62 @@
+body{
+    background-color: #9EA9FFFF;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.card-container{
+    display: flex;
+    grid-template-columns: repeat(2, 1fr);
```

Comment:
flex and grid template columns do not work hand in hand.

Reply:
'grid-template-columns' removed.